src/extensions/default/Inline3DParametersEditor/ParameterEditor.js

```diff
@@ -69,8 +69,7 @@ define(function (require, exports, module) {
     };
 
     ParameterEditor.prototype._setSpaces = function (parameters) {
-        this._spaces = [];
-        this._spaces.push(parameters.search(/\S/));
+        this._spaces = [parameters.search(/\S/)];
```

Comment:
The role of this function is to store p+1 number of spaces in the array _spaces.
Eg : "     2   3  3"
```js
this._spaces = [5, 3,  2, 0]
```
Eg : "2 3 1"
```js
this._spaces = [0, 1,  1, 0]
```
Eg : "2"
```js
this._spaces = [0, 0]
```
It is used to make sure that when parameters are updated, the number of spaces remain the same as before. See [this](https://github.com/mozilla/brackets/pull/651/files/4a546960cd9a18850f64b220c47c6b994722c289#diff-12ec682d75836bdc488d42abbe212a30R158)